docs/source/tutorial/01-configure-project.mdx

```diff
@@ -8,33 +8,34 @@ In this step, you'll import the starter project and add the Apollo Kotlin SDK to
 
 ```
 git clone https://github.com/apollographql/apollo-kotlin-tutorial
-cd apollo-kotlin-tutorial
-git checkout initial
+cd apollo-kotlin-tutorial/compose
 ```
 
-At any time, if you want to view a completed version of the project, checkout the `main` branch.
+The starter project is in the `initial` folder. 
+
+At any time, if you want to view a completed version of the project, open the project in the `finished` folder.
 
 ## Open the project in Android Studio
 
-Android Studio will take some time to index the contents of the project. Take some time to look around. You should see 3 Fragments:
+Android Studio will take some time to index the contents of the project. Take some time to look around. You should see 3 files:
 
-* `LaunchListFragment` is the first fragment. You will display a list of launches there.
-* `LaunchDetailsFragment` will display details about a launch and give you the opportunity to book a seat.
-* `LoginFragment` will allow you to log in before booking the seat.
+* `LaunchList.kt` is the first screen. You will display a list of launches there.
+* `LaunchDetails.kt` will display details about a launch and give you the opportunity to book a seat.
+* `Login.kt` will allow you to log in before booking the seat.
 
 Now build and run the project.
 
-Note: if you see build errors, make sure you are using JDK 11 or later.
+> **Note:** if you see build errors, make sure you are using JDK 11 or later.
```

Comment:
Can we add a small check in `build.gradle.kts` to make this foolproof and streamline the docs?